packaging/installer/README.md

```diff
@@ -165,3 +165,63 @@ the community helps fix any bugs that might have been introduced in previous rel
 -   Protect yourself from the rare instance when major bugs slip through our testing and negatively affect a Netdata
     installation
 -   Retain more control over the Netdata version you use
+
+## Installation notes and known issues
+
+We are tracking a few issues related to installation and packaging.
+
+### Failure to claim/connect to Netdata Cloud
+
+If you're running an older Linux distribution or one that has reached EOL, such as Ubuntu 14.04 LTS, Debian 8, or CentOS
+6/7, your Agent may not be able to connect to Cloud due to an old version of OpenSSL that is incompatible with Netdata
+Cloud.
+
+We recommend using the [static build installation](/packaging/installer/methods/kickstart-64.md) to avoid this issue.
+You can, however, use the workaround below.
+
+#### OpenSSL workaround
+
+Instead of using the static build, you can also reconfigure OpenSSL to allow self-signed certificates and skip hostname
+checks.
+
+> ⚠️ Using this workaround disables security checks in OpenSSL and reduces the security of your system and its metrics.
+> Allowing self-signed certificates and skipping hostname checks could allow a hostile intermediary to collect your
+> node's metrics as it transits to Netdata Cloud.
+
+If you choose to proceed, you must export the `-DACLK_SSL_ALLOW_SELF_SIGNED` CFLAG before installation.
+
+```bash
+export CFLAGS="-DACLK_SSL_ALLOW_SELF_SIGNED"
+```
+
+Then install Netdata using your method of choice.
+
+#### Ubuntu 14.04 LTS
+
+To use the `CFLAGS` workaround on Ubuntu 14.04 LTS, you must first install `libuv1-dev` via a PPA:
+
+```bash
+add-apt-repository ppa:acooks/libwebsockets6
+apt-get update
+apt-get install libuv1-dev
+```
+
+Then proceed with the CFLAGS workaround described above.
+
```

Comment:
This is already handled by #8916 not sure we need to duplicate the code in our docuemtnation as we don't do this for similar reasons for handling other systems.

Reply:
That PR is not merged yet. Is some other fix already present for Ubuntu 14.04?